man/ukify_hwid.json.example

```diff
@@ -1,4 +1,5 @@
 {
+    "devtype": "devicetree",
```

Comment:
"devtype" reads as if this was a type of device. but it's not, it's the type of resource you match agains the device. maybe just call this "type"?

Reply:
(of course, this is called devicetype inside our codebase, but maybe we shouldn't leak that into the json)